server/clip_server/model/model.py

```diff
@@ -111,6 +120,8 @@ class CLIPVisionCfg:
     patch_size: int = 16
     image_size: Union[Tuple[int, int], int] = 224
     ls_init_value: Optional[float] = None  # layer scale initial value
+    patch_dropout: float = 0.0  # what fraction of patches to dropout during training (0 would mean disabled and no patches dropped) - 0.5 to 0.75 recommended in the paper for optimal results
```

Comment:
why we need dropout during inference? It's not training process

Reply:
disabled by default

Reply:
Then we can remove this args if we don't need it